themes/jaeger-docs/static/js/latest-articles.js

```diff
@@ -0,0 +1,56 @@
+function get_latest_post(){
+  fetch('https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/jaegertracing')
+   .then((res) => res.json())
+   .then((data) => {
+      const res = data.items
+      const posts = res.filter(item => item.categories.length > 0)
+      // Functions to create a short text out of whole blog's content
+      function toText(node) {
+         let tag = document.createElement('div')
+         tag.innerHTML = node
+         node = tag.innerText
+         return node
+      }
+      function shortenText(text,startingPoint ,maxLength) {
+         return text.length > maxLength?
+         text.slice(startingPoint, maxLength):
+         text
+      }
+
+      // Put things in right spots of markup
+      let output = '';
+      for(var i = 0; i<posts.length; i++){
+        console.log(posts[i].link)
+         output += `
+         <li class="blog__post">
+            <a href="${posts[i].link}">
+
+               <img src="${posts[i].thumbnail}" class="blog__topImg"></img>
+
+
+               <div class="blog__content">
+                  <div class="blog_preview">
+                     <h2 class="blog__title">${shortenText(posts[i].title, 0, 30)+ '...'}</h2>
```

Comment:
nit: I would move `+ '...'` to `shortenText()` function and make it conditional on the text actually being longer than 30, otherwise you may add `...` when it's not needed